tests/vmi_hook_sidecar_test.go

```diff
@@ -303,17 +303,24 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 
 		Context("with ConfigMap in sidecar hook annotation", func() {
 
-			It("should update domain XML with SM BIOS properties", func() {
+			DescribeTable("[test_id:9833]should update domain XML with SM BIOS properties", func(withImage bool) {
```

Comment:
The commit is not reflecting that you added a new test. please refelct it in the commit or split the two and reason for each.